apps/passport-client/src/pcdPackages.ts

```diff
@@ -59,6 +61,12 @@ async function loadPackages(): Promise<PCDPackage[]> {
     zkeyFilePath: "/artifacts/zk-eddsa-event-ticket-pcd/circuit.zkey"
   });
 
+  await SecretPhrasePCDPackage.init({
+    verifyBaseURI: appConfig.secretPhraseClient,
```

Comment:
generally when adding new env vars, we either throw an error if it's undefined or provide a fallback